source/parse.h

```diff
@@ -1090,6 +1101,18 @@ struct template_argument
         std::unique_ptr<type_id_node>
     > arg;
 
+    // The type needs to be movable
+    // The copy ctor+operator are implicitly deleted due to the std::unique_ptr member
+    // Because a forward-declared type is used in a std::unique_ptr as a member an out-of-line dtor is necessary
+    // Because of the OOL dtor together with the fact that the copy ctor+operator are deleted
+    // the move ctor+operator need to be explicitly defaulted
+    // As a result the default constructor also needs to be explicitly defaulted
```

Comment:
This is an interesting side-effect of the changes.

Reply:
Good old SMF defaults 😁 Thanks!